modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -101,6 +103,9 @@ Liferay.Address = {
  */
 Liferay.DynamicSelect = DynamicSelect;
 
+Liferay.Disposable = Disposable;
+Liferay.EventEmitter = EventEmitter;
```

Comment:
I don't think we want to continue adding stuff to the `Liferay` global.

The global access pattern is a smell (one we've only tolerated in the past in the name of simplicity so that we can easily grab stuff from inside JSPs), and once we've added it, it is baggage we have to carry around potentially "forever".

So, if there are no consumers that would need this API, better not to have it. They can just use `import` instead.

Reply:
Yea, just like Chema said, I'll remove those and push.